src/card.css

```diff
@@ -0,0 +1,83 @@
+
+.card {
+  width: 200px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.imacImages {
+  width: 160px;
+  height: 134px;
+  margin: 32px 19px 30.5px;
+}
+
+.title {
+  font-family: Roboton, sans-serif;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 5px 16.02px 2px;
+  color: #060b35;
+}
+
+.subtitle {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 14px;
+  margin: 4.5px 16px;
+  color: #616070;
+}
+
+.reviews {
+  font-family: Roboto, sans-serif;
+  font-size: 10px;
+  line-height: 17px;
+  color: #060b35;
+}
+
+.price {
+  display: flex;
+  justify-content: space-between;
+  margin: 23px 16px 16px;
+}
+
+.textPrice {
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 12px;
+  line-height: 18px;
+  color: #616070;
+}
+
+.prices {
+  font-family: Robotos, sans-serif;
+  font-size: 16px;
+  line-height: 18px;
+  color: #060b35;
+}
+
+.button {
+  display: block;
+  background: rgb(0, 172, 220);
+  border-radius: 5px;
+  width: 166px;
+  height: 40px;
+  text-align: center;
+  color: rgb(255, 255, 255);
+  margin: 0 16px 16px;
+  line-height: 40.5px;
+  font-family: Robotos, sans-serif;
+  font-size: 14px;
+  text-transform: uppercase;
+  border: 1px solid #ccc transparent;
+}
+
+.button:hover {
+  cursor: pointer;
+  background-color: #fff;
+  color: #00acdc;
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  line-height: 39.5px;
```

Comment:
wow... why do u need 0.5px?

Reply:
because, without 0.5 px test doesn't pass(

Reply:
maybe we can use a different way of positioning